mediapipe/tasks/ios/vision/core/BUILD

```diff
@@ -61,11 +61,5 @@ objc_library(
         "//mediapipe/tasks/ios/core:MPPTaskRunner",
         "//third_party/apple_frameworks:UIKit",
         "@com_google_absl//absl/status:statusor",
-        "@ios_opencv//:OpencvFramework",
-    ] + select({
-        "@//third_party:opencv_ios_sim_arm64_source_build": ["@ios_opencv_source//:opencv_xcframework"],
-        "@//third_party:opencv_ios_sim_fat_source_build": ["@ios_opencv_source//:opencv_xcframework"],
-        "@//third_party:opencv_ios_arm64_source_build": ["@ios_opencv_source//:opencv_xcframework"],
-        "//conditions:default": [],
-    }),
+    ],
```

Comment:
@schmidt-sebastian This line of change somehow got reversed when the PR was imported. 
This is important for the vision pod to pass CocoaPods validation. I have raised a new PR which incorporates this.
https://github.com/google/mediapipe/pull/4386